assets/javascripts/discourse/initializers/checklist.js.es6

```diff
@@ -88,13 +88,15 @@ export function checklistSyntax($elem, post) {
             }
           );
 
-          cookAsync(newRaw).then(cooked =>
-            postModel.save({
-              cooked: cooked.string,
+          postModel
+            .save({
               raw: newRaw,
               edit_reason: I18n.t("checklist.edit_reason")
             })
-          );
+            .then(() => {
+              postWidget.attrs.isSaving = false;
```

Comment:
@eviltrout This is important. There is a race condition **somewhere**, where the widget was being rerendered while `isSaving` is true still... I tried, but I don't see what is happening. Manually setting `isSaving` to false before the re-render solves the issue, however.

Reply:
About 1/6 times a checkbox is clicked, the rerender happens out of order, and the post stalls out